src/main/java/com/epam/aidial/cfg/dao/jpa/GlobalInterceptorJpaRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.epam.aidial.cfg.dao.jpa;
+
+import com.epam.aidial.cfg.dao.model.GlobalInterceptorEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+import java.util.List;
+
+public interface GlobalInterceptorJpaRepository extends JpaRepository<GlobalInterceptorEntity, String> {
```

Comment:
epository extends JpaRepository<GlobalInterceptorEntity, String> expecting a String ID, but the database has a composite primary key, causing type mismatch

Reply:
no longer relevant